Modules/_ssl.c

```diff
@@ -5787,51 +5787,58 @@ sslmodule_add_option(PyObject *m, const char *name, uint64_t value)
 static int
 sslmodule_init_constants(PyObject *m)
 {
-    PyModule_AddStringConstant(m, "_DEFAULT_CIPHERS",
-                               PY_SSL_DEFAULT_CIPHER_STRING);
+    if (PyModule_AddStringConstant(m, "_DEFAULT_CIPHERS",
+                                   PY_SSL_DEFAULT_CIPHER_STRING) < 0) {
+        return -1;
+    }
+
+#define _PyModule_ADD_INT_CONST(module, name, value) \
+    if (PyModule_AddIntConstant(module, name, value) < 0) { \
+        return -1; \
+    }
```

Comment:
As in your other PR; please use the `do..while(0)` macro idiom:

```suggestion
#define _PyModule_ADD_INT_CONST(MODULE, NAME, VALUE) do {   \
    if (PyModule_AddIntConstant(MODULE, NAME, VALUE) < 0) { \
        return -1;                                          \
    }                                                       \
while (0)
```